src/main/java/fr/esgi/al5_2/Tayarim/controllers/AdministrateurController.java

```diff
@@ -0,0 +1,173 @@
+package fr.esgi.al5_2.Tayarim.controllers;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.exceptions.*;
+import fr.esgi.al5_2.Tayarim.services.AdministrateurService;
+import fr.esgi.al5_2.Tayarim.services.AuthService;
+import fr.esgi.al5_2.Tayarim.services.ProprietaireService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.ArrayList' import. Should be before 'org.springframework.web.bind.annotation.*'.